fiftyone/plugins/__init__.py

```diff
@@ -11,12 +11,14 @@
     enable_plugin,
     disable_plugin,
     delete_plugin,
+    delete_plugin_dir,
     list_plugins,
     list_disabled_plugins,
     list_downloaded_plugins,
     list_enabled_plugins,
     get_plugin,
     find_plugin,
+    find_duplicates,
```

Comment:
for consistency with other method names, suggest `find_duplicate_plugins` here

Reply:
updated